src/ashell/shell-state.c

```diff
@@ -578,6 +576,31 @@ int32_t ashell_check_control(const char *buf, uint32_t len)
     return RET_OK;
 }
 
+int32_t ashell_set_bootcfg(char *buf)
+{
+    if (!fs_valid_filename_size(buf)) {
+        return RET_ERROR;
+    }
+
+    fs_file_t *file = fs_open_alloc("boot.cfg", "w+");
+
```

Comment:
remove this blank line; ties the return value validation visually to getting the value